qml/Launcher/BackgroundBlur.qml

```diff
@@ -35,10 +35,7 @@ Item {
 
```

Comment:
Plz add additional copyright at beginning of file

Reply:
Added